src/Gui/QSint/actionpanel/actionpanelscheme.cpp

```diff
@@ -8,122 +8,108 @@
 #include "actionpanelscheme.h"
 
 
-class StaticLibInitializer
-{
-public:
-    StaticLibInitializer()
-    {
-        Q_INIT_RESOURCE(schemes);
-    }
-};
-
-StaticLibInitializer staticLibInitializer;
-
-
 namespace QSint
 {
 
 
-const char* ActionPanelDefaultStyle =
-
-    "QFrame[class='panel'] {"
-        "background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #99cccc, stop: 1 #EAF7FF);"
-    "}"
-
+const QString ActionPanelScheme::minimumStyle = QString::fromLatin1(
     "QSint--ActionGroup QFrame[class='header'] {"
-        "background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #F9FDFF, stop: 1 #EAF7FF);"
-        "border: 1px solid #00aa99;"
-        "border-bottom: 1px solid #99cccc;"
-        "border-top-left-radius: 3px;"
-        "border-top-right-radius: 3px;"
-    "}"
-
-    "QSint--ActionGroup QFrame[class='header']:hover {"
-        "background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #EAF7FF, stop: 1 #F9FDFF);"
+        "border: none;"
     "}"
 
     "QSint--ActionGroup QToolButton[class='header'] {"
-        "text-align: left;"
-        "font: 14px;"
-        "color: #006600;"
-        "background-color: transparent;"
-        "border: 1px solid transparent;"
+        "border: none;"
+        "font-weight: bold;"
+        "text-align: center;"
+        "background: none;"
     "}"
 
-    "QSint--ActionGroup QToolButton[class='header']:hover {"
-        "color: #00cc00;"
-        "text-decoration: underline;"
+    "QSint--ActionGroup QToolButton[class='action'] {"
+        "border: none;"
+        "background: none;"
     "}"
 
-    "QSint--ActionGroup QFrame[class='content'] {"
-        "background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #F9FDFF, stop: 1 #EAF7FF);"
-        "border: 1px solid #00aa99;"
+    "QSint--ActionGroup QToolButton[class='action']:hover {"
+        "text-decoration: underline;"
     "}"
 
     "QSint--ActionGroup QFrame[class='content'][header='true'] {"
-        "border-top: none;"
+        "border: none;"
     "}"
 
-    "QSint--ActionGroup QToolButton[class='action'] {"
-        "background-color: transparent;"
-        "border: 1px solid transparent;"
-        "color: #0033ff;"
-        "text-align: left;"
-        "font: 11px;"
-    "}"
+);
 
-    "QSint--ActionGroup QToolButton[class='action']:!enabled {"
-        "color: #999999;"
-    "}"
+QString ActionPanelScheme::systemStyle(const QPalette& p)
+{
+    const QColor& highlightColor = p.color(QPalette::Highlight);
+    QColor headerBackground = highlightColor.darker(150);
+    const QColor& groupBackground = p.color(QPalette::Button);
+
+    QHash<QString, QString> replacements;
+    replacements["headerBackground"] = headerBackground.name();
```

Comment:
Why don't we just hardcode the hex as previously?

Reply:
Because palette is dependent on stylesheet, hardcoded value is not.

Reply:
Good point. Though maybe we should  use string interpolation (`QString::arg`), rather than find and replace? @alfrix

Reply:
there were a lot of colors, keeping more than 10 colors was difficult to track by number
do not worry about this, all the stylesheet background colors need to be removed to allow auto light/dark transitions in classic theme